Tests/mock_server.py

```diff
@@ -267,6 +269,18 @@ def start(self, playbook_id, path=None, record=False):
         if self.process.returncode is not None:
             raise Exception("Proxy process terminated unexpectedly.\nExit code: {}\noutputs:\nSTDOUT\n{}\n\nSTDERR\n{}"
                             .format(self.process.returncode, self.process.stdout.read(), self.process.stderr.read()))
+        log_file_status = 2
+        counter = 0
+        # Make sure process is up and running
+        # While log file doesn't exist and less than PROXY_PROCESS_INIT_TIMEOUT seconds has passed
+        while log_file_status > 0 and counter < PROXY_PROCESS_INIT_TIMEOUT:
+            # Check if log file exist
+            log_file_status = self.ami.call(['ls', log_file])
```

Comment:
It might be nicer to have a boolean that tells us if the `log_file_exists`:
- Change `log_file_status` to `log_file_exists`
- init it to `False`
- set it to contain the (boolean) value of `self.ami.call(['ls', log_file]) == 0`

Reply:
- [x] Done